docs/modules/migrate/pages/upgrading-from-imdg-3.adoc

```diff
@@ -618,7 +618,7 @@ Hazelcast defaults to use OpenSSL when:
 
 * when you use xref:security:tls-ssl.adoc[TLS/SSL] and Hazelcast detects some
 xref:security:integrating-openssl.adoc[OpenSSL] capabilities
-* the Java version is less than 11
+* the Java version is less than 17
```

Comment:
This page is about migrating from IMDG 3 to Hazelcast 5.0.

- if it's really about 5.0, then it shouldn't change with the new 5.4 docs
- if it's actually about 5.x (which is _also_ used on this page) then I'm a bit confused, as you can't use <17 with 5.4?